extensions/ql-vscode/package.json

```diff
@@ -442,6 +442,18 @@
           "command": "codeQLDatabases.removeDatabase",
           "when": "false"
         },
+        {
+          "command": "codeQLDatabases.chooseDatabaseFolder",
```

Comment:
Do we want to disable these as global commands?

Reply:
We definitely *do* want to disable these commands if we want to consistently maintain the pattern that the hover text over the buttons is not be prefixed, and the command palette command names is prefixed with `CodeQL: `. And I think both of those patterns are desirable.

However, we totally could *also* have command-palette-visible commands (which would be named, like, `codeQL.chooseDatabaseFolder`) which do the same thing.

Reply:
That makes sense. Maybe we _don't_ want to pollute the global scope here with commands that are so tightly related to the databases view (I'm contradicting what I said earlier). So, what you have right now is fine.